src/Wallabag/CoreBundle/Resources/translations/messages.cs.yml

```diff
@@ -28,6 +28,7 @@ entry:
     page_titles:
         all: Všechny položky
         untagged: Položky bez štítků
+        # same_domain: Same domain
```

Comment:
You don't need to update all translation now. Just provide the English one and weblate will do the rest.

Reply:
Fixed.